tools/Makefile.am

```diff
@@ -3,7 +3,7 @@ sbin_PROGRAMS = mount.composefs
 noinst_PROGRAMS = 
 
 if USE_YAJL
-noinst_PROGRAMS += composefs-from-json
+noinst_PROGRAMS += composefs-from-json cfsfuse
```

Comment:
Naming schemes for filesystem tools is messy, but `cfsfuse` feels...well, I guess it's accurate, just kind of terse.  `mount.cfsfuse` maybe?  Dunno.

Reply:
`mount.*` is kinda reserved for /bin/mount helpers.

Other things seem be nonstandard, but mostly `xxx-fuse`:
* erofsfuse
* fuse2fs
* fuse-overlayfs
* rofiles-fuse
* vmhgfs-fuse   
* vmware-vmblock-fuse
* gvfsd-fuse 
* revokefs-fuse
* zfs-fuse-scrub
* zfs-fuse

I think i'll go with `composefs-fuse`, to match the existing `mount.composefs` and `mkcomposefs` tools.